terraform.tf.j2

```diff
@@ -258,6 +258,15 @@ data "aws_secretsmanager_secret_version" "secret_for_ris_collections" {
   secret_id = data.aws_secretsmanager_secret.secret_for_ris_collections.id
 }
 
+
```

Comment:
My understanding is that we usually use the bootstrap_terraform.py to populate variables within terraform.tfvars.json

Reply:
Looks like it is used elsewhere